dm/metrics/grafana/DM-Monitor-Professional.json

```diff
@@ -200,11 +200,11 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "dm_worker_task_state{task=~\"$task\",source_id=~\"$source\"}",
+          "expr": "dm_worker_task_state{job_id=~\"$job_id\",task=~\"$task\",source_id=~\"$source\"}",
           "format": "time_series",
           "interval": "$interval",
           "intervalFactor": 1,
-          "legendFormat": "{{task}}-{{worker}}",
+          "legendFormat": "{{task}}-{{source_id}}",
```

Comment:
value of work is different with source_id, 127.0.0.1:8262 vs mysql-01

Reply:
change lengend format to `{{task}}-{{source_id}}-{{worker}}`